packages/e2e-tests/puppeteer/helpers.ts

```diff
@@ -307,8 +317,12 @@ export const waitAndOpenChannel = (usingVirtualFunding: boolean) => async (
 export const waitForNthState = async (page: Page, n = 50): Promise<void> => {
   return doneWhen(page, `parseInt(channelStatus.turnNum) >= ${n}`);
 };
+export const waitForClosedState = async (page: Page): Promise<void> => {
+  return doneWhen(page, `channelStatus.status === 'closed'`);
+};
 
 export async function waitForClosingChannel(page: Page): Promise<void> {
+  await waitForWalletToBeDisplayed(page);
```

Comment:
Actually, this expectation should _not_ be here for virtual funding!

Reply:
I've created #1608 for this. Since this is how the `wallet` currently works I'll leave the check as is.